### DISCUSSION TITLE
Twitter news feed

### DISCUSSION THREAD

**Discussion by sepehr-alipour** [WeAreMahsaAmini/Core#28] [Ideas]
A mechanism to get the necessary news feed from Twitter to send as a group email or push to Telegram channel

**Comment by farnoodma** [comment]
@sepehr-alipour can you describe the difference between this feed and VahidOnline or Mamlekate channels? Basically, these two are gathering all the news mostly from Twitter and putting it inside their Telegram channels (with reference to the actual tweet).

**Reply by sepehr-alipour** [reply]
I think posts on those channels are publishing by admins while I'm thinking about pushing automatically by bots based on hashtags or specific accounts that people are willing to subscribe to.

**Comment by kaveh6** [comment]
Since right now mostly twitter is the source of news but people have easier access to Telegram , we could create a proxy in Telegram over twitter so people can follow accounts, write tweets and help with hashtag as well . And We can automatically postfix the tweets with hashtag .

**Reply by kaveh6** [reply]
but I think we would face twitter rate limiting mechanism , maybe we can tell them about the project and ask them to increase the limit at least for some time .
